src/python/pants/backend/python/target_types.py

```diff
@@ -466,6 +466,12 @@ class PythonTestsExtraEnvVars(StringSequenceField):
     )
 
 
+class SkipPythonTestsField(BoolField):
+    alias = "skip_tests"
+    default = False
+    help = "If true, don't run this target's tests."
```

Comment:
Not sure if it's worth mentioning why this is helpful? Like that the code can still be used with things like `./pants lint` and `./pants fmt`. Is that self-evident?